### PR TITLE
Bug 1909793: Backport: [Release 4.6] adds tunefastdeviceclass property

### PR DESCRIPTION
**Pull request by crombus**

### CODE REVIEW

**Comment by crombus** [comment]
Dependent on https://github.com/openshift/ocs-operator/pull/1021

File: pkg/controller/storagecluster/storagecluster_controller_test.go [hunk 1]
```diff
@@ -163,6 +163,55 @@ func TestNonWatchedResourceNameNotFound(t *testing.T) {
 	assert.Equal(t, reconcile.Result{}, result)
 }
 
+func TestThrottleStorageDevices(t *testing.T) {
```
Comment (obnoxxx) [inline pkg/controller/storagecluster/storagecluster_controller_test.go:166]: This function is actually added in previous commits. Shouldn't we cherry-pick those commits separately? The original commit had only small modifications to the test function.

**Review by obnoxxx — CHANGES_REQUESTED** [review]
There are a few hunks that don't belong to the original commit. Not sure we should merge like this - mainly in the test code so maybe?... 

Prelimiary request changes...

@jarrpa

**Comment by obnoxxx** [comment]
JFTR, @jarrpa agreed to require making this backport cleaner. So I'll leave the change request. @crombus

File: pkg/controller/storagecluster/storagecluster_controller_test.go (hunk 1 above)
Comment (crombus) [reply pkg/controller/storagecluster/storagecluster_controller_test.go:166]: yes and in release-4.6 the test function is not there https://github.com/openshift/ocs-operator/blob/release-4.6/pkg/controller/storagecluster/storagecluster_controller_test.go

Comment (crombus) [reply pkg/controller/storagecluster/storagecluster_controller_test.go:166]: the commit which adds the test is lost after the sdk update.

Comment (crombus) [reply pkg/controller/storagecluster/storagecluster_controller_test.go:166]: https://github.com/openshift/ocs-operator/pull/759/commits/2994cee1f5facd2d633be03035c89cfa38d30bbf
 found the commit

Comment (obnoxxx) [reply pkg/controller/storagecluster/storagecluster_controller_test.go:166]: The commit is not lost, but it is just on a different file than the test is in the current master: 2994cee1f5facd2d633be03035c89cfa38d30bbf

But this is actually too much different context.
I guess for this backport, it is ok to just add the function like this.
It *could* have been added as a separate commit to keep the actual commit to add TuneFastDeviceClass closer to the original, but I won't insist.

**Review by obnoxxx — APPROVED** [review]
Looks clean now, thanks for the update, @crombus .

As discussed, we can add the test function like this, because the history is too convoluted.

/lgtm

**Pull request merged**